src/config/constants.js

```diff
@@ -0,0 +1,9 @@
+
+module.exports = {
+  auth0: {
+    DOMAIN: process.env.AUTH0_DOMAIN,
+    CLIENT_ID: process.env.AUTH0_CLIENT_ID,
+    CLIENT_SECRET: process.env.AUTH0_CLIENT_SECRET,
+    CERTIFICATE: process.env.AUTH0_CERTIFICATE,
```

Comment:
These data is defined in azure as `Application Settings`, everything works great except for `AUTH0_CERTIFICATE`. For now I hardcoded the value of this variable in the azure console, which is not ideal.

The value of this variable is something like:
```
AUTH0_CERTIFICATE = `
-----BEGIN CERTIFICATE-----
ABC123.....
ABC123.....
ABC123.....
-----END CERTIFICATE-----`
```

It looks like when saving this value in the applications settings azure removes the `\n` and when used in the code doesn't work :(

Anyone can help me with this?